src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static ArrayList<Task> list = new ArrayList<Task>();
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            String input = in.nextLine();
+            if(input.equalsIgnoreCase("bye")) {
+                bye();
+                break;
+            }
+            processInput(input);
+        }
     }
+    public static void greet() {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+    }
+    public static void bye() {
```

Comment:
Perhaps you can consider leaving one line of space between different methods to make it more clear where one method ends and the next begins.